relay-general/src/protocol/span.rs

```diff
@@ -18,7 +18,7 @@ pub struct Span {
     pub exclusive_time: Annotated<f64>,
 
     /// Human readable description of a span (e.g. method URL).
-    #[metastructure(pii = "maybe")]
+    #[metastructure(pii = "true")]
```

Comment:
`data` field is also `maybe` and some SDKs also set the URL in there.
See this https://github.com/getsentry/sentry/blob/fef9c695a1a7d3384fb3ce7ec6c264632e77061d/static/app/components/events/interfaces/spans/spanTreeModel.spec.tsx#L47
and https://github.com/getsentry/sentry/blob/fef9c695a1a7d3384fb3ce7ec6c264632e77061d/src/sentry/interfaces/spans.py#L33-L38

Reply:
yep I  will add this